src/components/userAuth/signInMail/forgotPassword/resetPassword/PasswordHintColorAndIcon.tsx

```diff
@@ -0,0 +1,95 @@
+import { Box, FormHelperText, List, ListItem, Typography } from '@mui/material'
+import { makeStyles } from 'tss-react/mui'
+
+const PasswordHintColorAndIcon = ({ password }: { password: string }) => {
+  const { classes } = useStyles()
+
+  const passwordHintColor = (pattern: RegExp, password: string) => {
+    if (password) {
+      if (pattern.test(password)) {
+        return '#1D878C'
+      }
+      return '#F46B5D'
+    }
+    return '#444444'
+  }
+
+  const passwordHintIcon = (pattern: RegExp, password: string) => {
+    if (password && pattern.test(password)) {
+      return '/img/check.svg'
+    }
+    return '/img/hyphen.svg'
+  }
+
+  return (
+    <FormHelperText component="div" className={classes.errorBox}>
+      <Typography
+        sx={{ color: '#444444', fontSize: '14px', lineHeight: '22px' }}
+      >
+        Your Password must have:
+      </Typography>
+      <List>
+        <ListItem
+          sx={{
+            color: passwordHintColor(/.{8,}/, password),
+            padding: 0,
+          }}
+        >
+          <Box component="img" src={passwordHintIcon(/.{8,}/, password)}></Box>
+          &nbsp;8 or more symbols
+        </ListItem>
+        <ListItem
+          sx={{
+            color: passwordHintColor(/[0-9]/, password),
+            padding: 0,
+          }}
+        >
+          <Box component="img" src={passwordHintIcon(/[0-9]/, password)}></Box>
+          &nbsp;1 or more numbers
+        </ListItem>
+        <ListItem
+          sx={{
+            color: passwordHintColor(/[a-zA-Z]/, password),
```

Comment:
move regular expression to const on the top

Reply:
done